### PR TITLE
Implement IMetricLogger

### PR DESCRIPTION
**Pull request by chrisproud**
Here is the implementation of the basic abstraction IMetricLogger as described in https://github.com/GlobeBMG/GBMG.Monitoring/issues/1.

### CODE REVIEW

File: source/GBMG.Monitoring/GBMG.Monitoring.csproj [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="interfaces\IMetricLogger.cs" />
```
Comment (danbrad) [inline source/GBMG.Monitoring/GBMG.Monitoring.csproj]: Why lower case i ?

File: source/GBMG.Monitoring/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("GBMG monitoring and telemetry abstraction")]
+[assembly: AssemblyDescription("A generic interface for sending custom metrics and telemetry to the configured monitoring provider")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Globe Business Media Group")]
+[assembly: AssemblyProduct("GBMG.Monitoring")]
+[assembly: AssemblyCopyright("Copyright ©  2016")]
```
Comment (danbrad) [inline source/GBMG.Monitoring/Properties/AssemblyInfo.cs]: Copyright © Globe Business Media Group 2016

**Comment by chrisproud** [comment]
I've made those amends. When your happy can you merge and close?

**Comment by msancho** [comment]
Will do tomorrow morning, I want to check something ta.

**Pull request merged**